rmgpy/cantherm/molepro.py

```diff
@@ -42,7 +45,174 @@ class MoleProLog:
+                    count += 1
+                    line = f.readline()
+            line = f.readline()
+        # Close file when finished
+        f.close()
+
+        coord = numpy.array(coord, numpy.float64)
+        number = numpy.zeros(len(symbol), numpy.int)
+        mass = numpy.zeros(len(symbol), numpy.float64)
+        # Use the atomic mass of the most common isotope rather than the
+        # average atomic mass
+        # These values were taken from "Atomic Weights and Isotopic Compositions" v3.0 (July 2010) from NIST
+        for i in range(len(symbol)):
+            if symbol[i] == 'H':
+                number[i] = 1
+                mass[i] = 1.00782503207
+            elif symbol[i] == 'C':
+                number[i] = 6
+                mass[i] = 12.0
+            elif symbol[i] == 'N':
+                number[i] = 7
+                mass[i] = 14.0030740048
+            elif symbol[i] == 'O':
+                number[i] = 8
+                mass[i] = 15.99491461956
+            elif symbol[i] == 'P':
+                number[i] = 15
+                mass[i] = 30.97376163
+            elif symbol[i] == 'S':
+                number[i] = 16
+                mass[i] = 31.97207100
+            elif symbol[i] == 'Cl':
+                number[i] = 17
+                mass[i] = 35.4527
+            elif symbol[i] == 'I':
+                number[i] = 53
+                mass[i] = 126.90447
+            else:
+                print 'Atomic number {0:d} not yet supported in loadGeometry().'.format(number[i])
+        return coord, number, mass
+
+    def loadConformer(self, symmetry=None, spinMultiplicity=None, opticalIsomers=1):
+        """
+        Load the molecular degree of freedom data from a log file created as
+        the result of a MolPro "Freq" quantum chemistry calculation with the thermo printed.
+        """
+
+        modes = []
+        E0 = 0.0
+
+        f = open(self.path, 'r')
+        line = f.readline()
+        while line != '':
+
+            # The data we want is in the Thermochemistry section of the output
+            if 'THERMODYNAMICAL' in line:
+                modes = []
+                inPartitionFunctions = False
+                line = f.readline()
+                while line != '':
+
+                    # This marks the end of the thermochemistry section
+                    if '*************************************************' in line:
+                        break
+
+                    # Read molecular mass for external translational modes
+                    elif 'Molecular Mass:' in line:
+                        mass = float(line.split()[2])
+                        translation = IdealGasTranslation(mass=(mass,"amu"))
+                        modes.append(translation)
+                    # Read MolPro's estimate of the external symmetry number
+                    elif 'Rotational Symmetry factor' in line and symmetry is None:
+                        symmetry = int(float(line.split()[3]))
```

Comment:
The next version will use the value put into the input file if there is one. If there is not one it will default to use the symmetry estimated by the software.